src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers, createStore } from 'redux';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
```

Comment:
- [ ] Kindly, import and combine the `categoriesReducer` as instructed in the requirements.

Reply:
Thank you for the review @omar-labana I have resolved it.